source/braille.py

```diff
@@ -2777,10 +2777,11 @@ def setDisplayByName(
 				elif (
 					"bluetoothName" in detected.deviceInfo
 					or detected.deviceInfo.get("provider") == "bluetooth"
+					or detected.deviceInfo.get("provider") == "ble"
```

Comment:
I think you can merge this line and the line above
```suggestion
					or detected.deviceInfo.get("provider") in "("bluetooth", "ble")
```